ui/src/components/common/Breadcrumbs/index.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { Breadcrumbs } from "./index";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+
+let mockPATH = "";
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useLocation: () => ({
+    pathname: mockPATH,
+  }),
+}));
+
+describe("Breadcrumbs", () => {
```

Comment:
we can have one describe section with multiple it blcoks

Reply:
will move `it` under a single block